packages/flutter_tools/lib/src/commands/fuchsia_reload.dart

```diff
@@ -355,11 +362,15 @@ class FuchsiaReloadCommand extends FlutterCommand {
     return <String>[path, name];
   }
 
-  Future<List<_PortForwarder>> _forwardPorts(List<int> remotePorts) {
+  Future<List<_PortForwarder>> _forwardPorts(List<int> remotePorts) async {
     final String config = '$_buildDir/ssh-keys/ssh_config';
-    return Future.wait(remotePorts.map((int remotePort) {
-      return _PortForwarder.start(config, _address, remotePort);
-    }));
+    final List<_PortForwarder> forwarders = <_PortForwarder>[];
+    for (int port in remotePorts) {
+      final _PortForwarder f =
+          await _PortForwarder.start(config, _address, port);
```

Comment:
Consider accumulating futures and using Future.wait for the result so the tunnels can start in parallel.